zcobol/README.TXT

```diff
@@ -1,7 +1,20 @@
 .*********************************************************************
-.* Copyright 2009 Automated Software Tools Corporation               *
-.* This source code is part of z390 zCOBOL/assembler/emulator package*
-.* The z390 package is distributed under GNU general public license  *
+.* z390 - Mainframe assembler emulator and run-time engine
+.* Copyright (C) 2021 z390 Assembler LLC
+.*
+.* This file is part of z390.
+.*
+.* z390 is free software; you can redistribute it and/or modify
+.* it under the terms of the GNU General Public License as published by
+.* the Free Software Foundation; either version 2 of the License, or
+.* (at your option) any later version.
+.* z390 is distributed in the hope that it will be useful,
+.* but WITHOUT ANY WARRANTY; without even the implied warranty of
+.* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+.* GNU General Public License for more details.
+.*
+.* You should have received a copy of the GNU General Public License 
+.* along with this program; if not, see https://www.gnu.org/licenses.
 .* Author - Don Higgins                                              *
 .* Date   - 02/20/09                                                 *
 .*********************************************************************
```

Comment:
Suggestion only. README.TXT is a text file, not a macro file. Replace the leading ".*" with "*". Yes, it already had the leading ".*".  IMO, that's "wrong".

Reply:
Fixed as suggested